From 0a2bf065882fa3bf823a2483c6b6f93195a08922 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Mon, 13 Jun 2011 16:31:11 +0100
Subject: [PATCH] Don't version the panel plugin

---
 src/panel/Makefile.am |    6 ++++--
 1 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/src/panel/Makefile.am b/src/panel/Makefile.am
index 4383b3d..2de2865 100644
--- a/src/panel/Makefile.am
+++ b/src/panel/Makefile.am
@@ -30,14 +30,16 @@ libgoa_panel_la_CFLAGS =					\
 	$(GNOME_CONTROL_CENTER_CFLAGS)				\
 	$(NULL)
 
-libgoa_panel_la_LDFLAGS = 					\
-	`pkg-config --libs webkitgtk-3.0` \
+libgoa_panel_la_LDFLAGS = 			\
+	-avoid-version -no-undefined		\
+	-export-symbols-regex '^g_io_module_(load|unload)'
 	$(NULL)
 
 libgoa_panel_la_LIBADD = 					\
 	$(GLIB_LIBS) 						\
 	$(GTK_LIBS)						\
 	$(GNOME_CONTROL_CENTER_LIBS)				\
+	$(WEBKITGTK_LIBS)					\
 	$(top_builddir)/src/goa/libgoa-1.0.la			\
 	$(top_builddir)/src/goabackend/libgoa-backend-1.0.la	\
 	$(NULL)
-- 
1.7.5.4

